Feat(eos_designs,eos_cli_config_gen)!: Change default encapsulation to path-selection for WAN iBGP EVPN peerings (#4496)
aristanetworks/ansible-avd: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

      - AWE-7220R
```

#### Default encapsulation for WAN EVPN iBGP peerings changed to `path-selection`

> With AVD version 5.0.0 the encapsulation used by default for EVPN iBGP peering for WAN router is `path-selection`. Before AVD 5.x.x only the default encapsulation (`vxlan`) was supported.

Reviewer: ```suggestion
With AVD version 5.0.0 the default encapsulation used for EVPN iBGP peering between WAN routers is `path-selection`. Previously it was `vxlan`.
```

> 

Reviewer: ```suggestion

!!! danger
    The `path-selection` encapsulation is only supported on EOS above 4.33.0F.

    It is critical that all WAN devices use the same encapsulation. For existing deployments it is recommended to set the encapsulation back to `vxlan` until a migration can be done across the entire WAN network.

    Contact your Arista account team for guidance on migration.
```

Reviewer: maybe this shoudl also go into the release notes.
